tests/reg_tests/test_adjoint.py

```diff
@@ -75,12 +75,11 @@ def span(val, geo):
             "l2convergence": 1e-14,
             "monitorvariables": ["cpu", "resrho", "totalr"],
             "adjointl2convergence": 1e-14,
-            "mgcycle": "2w",
+            "mgcycle": "sg",
```

Comment:
Do we need to start with ANK for all the complex tests? I know that you added the solver combos test, but it would be nice to still have DADI/RK as part of a complex regression test (not just convergence).

Reply:
thats fair, I guess I wanted to see if everything would still work that way. To test it even more, I can go back to smoother+MG and then switch to ank and then to nk. That way tests touch more solvers basically. Does that make sense @sseraj ?

Reply:
Yes, that would be great.